messages app - update layout to allow the messages within viewport

achen2401 opened the pull request:
https://www.pivotaltracker.com/story/show/184838934
Changes include:
- re-arrange components within the messages app to allow the messages be visible within viewport without scrolling.
- re-formatting code
- change position of header banner to `fixed` for it to remain at top when scrolling.
- added patient demographic info, i.e. name and dob at header banner for clinical safety

Example screenshot after changes:
desktop view:
<img width="1244" alt="Screen Shot 2023-03-31 at 5 23 44 PM" src="https://user-images.githubusercontent.com/12942714/229256283-98d96aab-a864-4eb2-950a-1a0214a236f1.png">

mobile view:
<img width="490" alt="Screen Shot 2023-03-31 at 5 20 17 PM" src="https://user-images.githubusercontent.com/12942714/229256300-0184e7e1-c69c-48eb-bd10-634f8c34573e.png">

Commit:
messages app - rearrange layout to make messsages view within viewport

mcjustin commented:
Just capturing a couple screenshots of the existing messaging client for posterity.

Full screen desktop, zoomed out so it all fits in:
![image](https://user-images.githubusercontent.com/1036500/229248756-2db62e7b-42d2-4191-8886-dbae8126d0e4.png)

Phone at load:
![image](https://user-images.githubusercontent.com/1036500/229248862-f66e3f6d-b851-4655-9374-2c6f5bbb588d.png)

Phone scrolled to bottom:
![image](https://user-images.githubusercontent.com/1036500/229248971-46344e4b-4022-4f82-b404-79a143cf9a9e.png)

mcjustin commented:
@achen2401 I think we need a floating patient demographics banner that always remains the top, like we do on other systems - for clinical safety. This is most important in phone layout - after you've made that change can you please share a screenshot in phone layout?

Commit:
fixed header to allow display of patient info

achen2401 commented:
> @achen2401 I think we need a floating patient demographics banner that always remains the top, like we do on other systems - for clinical safety. This is most important in phone layout - after you've made that change can you please share a screenshot in phone layout?

@mcjustin thanks, I added the patient demo info to the header banner (now fixed and remains at top) and have attached the updated screenshots in the description.

Commit:
Merge branch 'dev' of https://github.com/uwcirg/isacc-messaging-client-sof into feature/messages-view-layout-update

Commit:
bug fix

Commit:
remove console.log statement

Commit:
Merge branch 'dev' of https://github.com/uwcirg/isacc-messaging-client-sof into feature/messages-view-layout-update
